sycl/test/graph/graph-explicit-multiple-exec-graphs.cpp

```diff
@@ -0,0 +1,100 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Is this PR enough to satisfy https://github.com/reble/llvm/issues/17 ?

Reply:
I believe so, I've updated the description to close that issue on merging.